nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -291,111 +501,131 @@ def permuted_ols(tested_vars, target_vars, confounding_vars=None,
+        ``model_intercept``).
+
+    model_intercept : :obj:`bool`, optional
+        If True, a constant column is added to the confounding variates
+        unless the tested variate is already the intercept.
+        Default=True.
+
+    masker
+
+    cdt : :obj:`float`, optional
+        Cluster-defining threshold, for cluster-level FWE correction.
+        The threshold should be a minimum t-statistic.
+
+    n_perm : :obj:`int`, optional
+        Number of permutations to perform.
+        Permutations are costly but the more are performed, the more precision
+        one gets in the p-values estimation. Default=10000.
+
+    two_sided_test : :obj:`bool`, optional
+        If True, performs an unsigned t-test. Both positive and negative
+        effects are considered; the null hypothesis is that the effect is zero.
+        If False, only positive effects are considered as relevant. The null
+        hypothesis is that the effect is zero or negative. Default=True.
+
+    random_state : :obj:`int` or None, optional
+        Seed for random number generator, to have the same permutations
+        in each computing units.
+
+    n_jobs : :obj:`int`, optional
+        Number of parallel workers.
+        If 0 is provided, all CPUs are used.
+        A negative number indicates that all the CPUs except (abs(n_jobs) - 1)
+        ones will be used. Default=1.
+
+    verbose : :obj:`int`, optional
         verbosity level (0 means no message). Default=0.
 
     Returns
     -------
-    pvals : array-like, shape=(n_regressors, n_descriptors)
-      Negative log10 p-values associated with the significance test of the
-      n_regressors explanatory variates against the n_descriptors target
-      variates. Family-wise corrected p-values.
+    neg_log10_pvals : array-like, shape=(n_regressors, n_descriptors)
+        Negative log10 p-values associated with the significance test of the
+        ``n_regressors`` explanatory variates against the ``n_descriptors``
+        target variates. Family-wise corrected p-values.
 
     score_orig_data : numpy.ndarray, shape=(n_regressors, n_descriptors)
-      t-statistic associated with the significance test of the n_regressors
-      explanatory variates against the n_descriptors target variates.
-      The ranks of the scores into the h0 distribution correspond to the
-      p-values.
+        T-statistics associated with the significance test of the
+        ``n_regressors`` explanatory variates against the ``n_descriptors``
+        target variates.
+        The ranks of the scores into the h0 distribution correspond to the
+        p-values.
 
-    h0_fmax : array-like, shape=(n_perm, )
-      Distribution of the (max) t-statistic under the null hypothesis
-      (obtained from the permutations). Array is sorted.
+    h0_fmax : array-like, shape=(n_regressors, n_perm)
+        Distribution of the (max) t-statistic under the null hypothesis
+        (obtained from the permutations). Array is sorted.
```

Comment:
This change is related to #3180. I will probably revert it here, but it's worth noting.

Reply:
Resolving this and reverting the shape documentation change in favor of discussing in #3180.